lib/iex/lib/iex/helpers.ex

```diff
@@ -89,7 +89,10 @@ defmodule IEx.Helpers do
       Code.delete_path(consolidation)
       purge_protocols(consolidation)
 
-      {result, _} = Mix.Task.run("compile")
+      force? = Keyword.fetch!(options, :force)
```

Comment:
You want to do `Keyword.get(options, :force, false)` otherwise `recompile!([])` will raise. This way you can set the default to an empty list.